teb_local_planner/package.xml

```diff
@@ -46,4 +45,4 @@
   <export>
     <build_type>ament_cmake</build_type>
```

Comment:
Your missing the pluginlib export, see this for reference https://github.com/ros-planning/navigation2/blob/master/nav2_navfn_planner/package.xml#L37

Reply:
Fixed.